natlas-server/app/instrumentation/__init__.py

```diff
@@ -47,15 +47,15 @@ def initialize_opentelemetry(config: Config, flask_app: flask.Flask) -> None:
 
 
 def initialize_sentryio(config: Config) -> None:
-    if config.sentry_dsn:
```

Comment:
I don't love the screaming snake case but it seems like that's the Flask norm, and Flask's config loader only loads things that are screaming snake case, so just to keep it consistent, I made the changes.